axelor-business-project/src/main/resources/domains/Project.xml

```diff
@@ -42,9 +42,11 @@
 	  	<one-to-many name="expensesLineList" ref="com.axelor.apps.hr.db.ExpenseLine" title="Expense lines" mappedBy="project" orphanRemoval="false"/>
 		<one-to-many name="invoiceLineList" ref="com.axelor.apps.account.db.InvoiceLine" title="Invoice lines" mappedBy="project" orphanRemoval="false"/>
 		
-		
 		<boolean name="isBusinessProject" title="Business project" />
 
+		<boolean name="invoicingExpences"/>
+		<boolean name="invoicingPurchaces"/>
```

Comment:
Where is title ? Also follow codding guidelines to start boolean field name with 'is'

Reply:
Purchases
Expenses